www/src/examples/Dropdown/Dropup.js

```diff
@@ -1,26 +0,0 @@
-<>
```

Comment:
This was unused. It seemed pointless because we already have the "directions" example.